k12/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["cryptography", "no-std"]
 
 [dependencies]
 digest = { version = "0.10", features = ["alloc"] }
+keccak = { git = "https://github.com/aewag/sponges", branch = "1-add-keccak-p" }
```

Comment:
This needs to be changed in the future. `keccak-p` is not yet available in the official keccak crate.